runners/direct-java/src/main/java/org/apache/beam/runners/direct/portable/WindowEvaluatorFactory.java

```diff
@@ -43,41 +42,32 @@
 
   @Override
   public <InputT> TransformEvaluator<InputT> forApplication(
-      AppliedPTransform<?, ?, ?> application,
+      PTransformNode application,
       @Nullable CommittedBundle<?> inputBundle
  )
       throws Exception {
-    return createTransformEvaluator((AppliedPTransform) application);
+    return createTransformEvaluator((PTransformNode) application);
```

Comment:
I don't think you need this case anymore since you don't have any wildcard types anymore.

Reply:
Removed.